content/rhoai/rosa-gpu/index.md

```diff
@@ -0,0 +1,380 @@
+
+This guide is a simple example on how to run and deploy LLMs on a [Red Hat OpenShift Services on AWS (ROSA)](https://www.redhat.com/en/technologies/cloud-computing/openshift/aws) cluster, which is our managed service OpenShift platform on AWS, using [Red Hat OpenShift AI (RHOAI)](https://www.redhat.com/en/technologies/cloud-computing/openshift/openshift-ai), which is formerly called Red Hat OpenShift Data Science (RHODS) and is our OpenShift platform for managing the entire lifecycle of AI/ML projects. And we will utilize [Amazon S3](https://aws.amazon.com/s3/) bucket to store the model output. In essence, here we will first install RHOAI operator and Jupyter notebook, create the S3 bucket, and then run the model.
+
+*Please note that the UI may change from time to time so what you see in the snippets below might change as well.*
+-->
+
+## Prerequisites
+
+### Tools
+
+* [AWS CLI](https://aws.amazon.com/cli/)
+* [OpenShift CLI](https://docs.openshift.com/container-platform/4.14/cli_reference/openshift_cli/getting-started-cli.html)
+* [ROSA CLI]()
+
+### Environment
+
+1. You will need a ROSA cluster, if you don't have one, you can follow the [ROSA guide](/experts/rosa/terraform/hcp) to create a ROSA cluster.
+
+2. You will need a GPU enabled machine pool in your ROSA cluster. If you don't have one, you can follow the [Adding GPUs to a ROSA cluster](/experts/rosa/gpu) guide to add GPUs to your cluster.
+
+## Install OpenShift Service Mesh Operator
+
+1. Deploy the Operator
+
+    ```bash
+    cat << EOF | oc apply -f -
+    apiVersion: operators.coreos.com/v1alpha1
+    kind: Subscription
+    metadata:
+      name: servicemeshoperator
+      namespace: openshift-operators
+    spec:
+      channel: stable
+      installPlanApproval: Automatic
+      name: servicemeshoperator
+      source: redhat-operators
+      sourceNamespace: openshift-marketplace
+    EOF
+    ```
+
+## Installing RHOAI and Jupyter notebook
+
+1. Create a project for the RHOAI operator:
+
+    ```bash
+    oc new-project redhat-ods-operator
+    ```
+
+1. Deploy the OpenShift AI Operator:
+
+    ```yaml
+    cat << EOF | oc apply -f -
+    apiVersion: operators.coreos.com/v1
+    kind: OperatorGroup
+    metadata:
+      name: redhat-ods-operator
+      namespace: redhat-ods-operator
+    spec:
+      upgradeStrategy: Default
+    ---
+    apiVersion: operators.coreos.com/v1alpha1
+    kind: Subscription
+    metadata:
+      name: rhods-operator
+      namespace: redhat-ods-operator
+    spec:
+      channel: fast
+      installPlanApproval: Automatic
+      name: rhods-operator
+      source: redhat-operators
+      sourceNamespace: openshift-marketplace
+      startingCSV: rhods-operator.2.9.1
+    EOF
+    ```
```

Comment:
tried this just now and it gave me bunch of error message where in essence the CSV isn't correct, we do not have rhods-operator-2.9.1 on fast channel as of now